[2023-06-10T01:42:09.351+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: table_check.load_data manual__2023-06-10T01:42:01.447311+00:00 [queued]>
[2023-06-10T01:42:09.358+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: table_check.load_data manual__2023-06-10T01:42:01.447311+00:00 [queued]>
[2023-06-10T01:42:09.358+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2023-06-10T01:42:09.370+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): load_data> on 2023-06-10 01:42:01.447311+00:00
[2023-06-10T01:42:09.375+0000] {standard_task_runner.py:57} INFO - Started process 48730 to run task
[2023-06-10T01:42:09.380+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'table_check', 'load_data', 'manual__2023-06-10T01:42:01.447311+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/schedule_job_dag.py', '--cfg-path', '/tmp/tmpcv9_jd4t']
[2023-06-10T01:42:09.382+0000] {standard_task_runner.py:85} INFO - Job 14: Subtask load_data
[2023-06-10T01:42:09.414+0000] {task_command.py:410} INFO - Running <TaskInstance: table_check.load_data manual__2023-06-10T01:42:01.447311+00:00 [running]> on host database
[2023-06-10T01:42:09.464+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='mail@hungtuan.me' AIRFLOW_CTX_DAG_OWNER='Tuan Nguyen' AIRFLOW_CTX_DAG_ID='table_check' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2023-06-10T01:42:01.447311+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-06-10T01:42:01.447311+00:00'
[2023-06-10T01:42:09.465+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-06-10T01:42:09.465+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python /root/table_check_project/etl_processes/load_data.py']
[2023-06-10T01:42:09.476+0000] {subprocess.py:86} INFO - Output:
[2023-06-10T01:42:09.973+0000] {subprocess.py:93} INFO - Data loaded successfully. Rows affected: 150000
[2023-06-10T01:42:10.042+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-06-10T01:42:10.057+0000] {taskinstance.py:1345} INFO - Marking task as SUCCESS. dag_id=table_check, task_id=load_data, execution_date=20230610T014201, start_date=20230610T014209, end_date=20230610T014210
[2023-06-10T01:42:10.111+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-06-10T01:42:10.123+0000] {taskinstance.py:2651} INFO - 0 downstream tasks scheduled from follow-on schedule check
